src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -68,6 +73,16 @@ public function getUpgradeStrategy(): string
         return (string)getenv('UPGRADE_STRATEGY') ?: static::RELEASE_APP_STRATEGY;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @return bool
+     */
+    public function getComposerInstallDependencies(): bool
+    {
+        return (bool)getenv('COMPOSER_INSTALL_DEPENDENCIES') ?: static::COMPOSER_INSTALL_DEPENDENCIES;
```

Comment:
Please add information about the configuration `COMPOSER_INSTALL_DEPENDENCIES` to README.md

Reply:
+